### ISSUE TITLE
Data error on graph on dashboard

### ISSUE THREAD

**Issue by EmilieBy** [glpi-project/glpi#15708]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

10.0.6 and 10.0.9

### Bug description

There is a error on association of wording and data.
I have 4 tickets assigned to Logistic Unit, 1 ticket assigned to Immobilier Unit, 2 tickets for Marché unit
On the graph, we can see 4 tickets for Immobilier Unit, 1 ticket for Logistic Unit and only 1 ticket for Marché unit

![image](https://github.com/glpi-project/glpi/assets/119668416/7868fb2e-def6-4da9-9b80-68337e890cd8)

![image](https://github.com/glpi-project/glpi/assets/119668416/5537043c-b40d-4cd8-8046-0b011e433024)

And there is a spelling mistake on the graph's title.
Ticket par statut SLA et paR groupe de techniciens

My issue #15582 was closed but i have the same problem in v10.0.9!!
And a spelling mistake on the graph's title is always exist!!!

Thanks for correction

### Relevant log output

```shell
<details><summary>Informations sur le système, l'installation et la configuration</summary><pre>GLPI 10.0.9 (/glpioscarr => /var/www/html/glpioscarr)
Installation mode: TARBALL
Current language:fr_FR

</pre></details><details><summary>Server</summary><pre> 
Operating system: Linux app116901l372.cpam-rhone.ramage 3.10.0-1160.el7.x86_64 #1 SMP Mon Oct 19 16:18:59 UTC 2020 x86_64
PHP 8.1.16 apache2handler (Core, PDO, Phar, Reflection, SPL, SimpleXML, Zend OPcache, apache2handler, bcmath, bz2, calendar,
	ctype, curl, date, dom, exif, fileinfo, filter, ftp, gd, gettext, hash, iconv, igbinary, intl, json, ldap, libxml, mbstring,
	memcache, msgpack, mysqli, mysqlnd, openssl, pcre, pdo_mysql, pdo_sqlite, posix, redis, session, shmop, sockets, sodium,
	sqlite3, standard, sysvmsg, sysvsem, sysvshm, tokenizer, xml, xmlreader, xmlwriter, xsl, zip, zlib)
Setup: max_execution_time="30" memory_limit="128M" post_max_size="8M" safe_mode="" session.save_handler="files"
	upload_max_filesize="2M" 
Software: Apache/2.4.34 (Red Hat) PHP/8.1.16 ()
	Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:102.0) Gecko/20100101 Firefox/102.0
Server Software: MariaDB Server
	Server Version: 10.5.16-MariaDB
	Server SQL Mode: STRICT_TRANS_TABLES,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION
	Parameters: root@localhost/glpioscarr
	Host info: Localhost via UNIX socket
	
PHP version (8.1.16) is supported.
Sessions configuration is OK.
Allocated memory is sufficient.
mysqli extension is installed.
Following extensions are installed: dom, fileinfo, filter, libxml, json, simplexml, xmlreader, xmlwriter.
curl extension is installed.
gd extension is installed.
intl extension is installed.
zlib extension is installed.
The constant SODIUM_CRYPTO_AEAD_XCHACHA20POLY1305_IETF_NPUBBYTES is present.
Database engine version (10.5.16) is supported.
No files from previous GLPI version detected.
The log file has been created successfully.
Write access to /var/www/html/glpioscarr/files/_cache has been validated.
Write access to /var/www/html/glpioscarr/config has been validated.
Write access to /var/www/html/glpioscarr/files/_cron has been validated.
Write access to /var/www/html/glpioscarr/files has been validated.
Write access to /var/www/html/glpioscarr/files/_dumps has been validated.
Write access to /var/www/html/glpioscarr/files/_graphs has been validated.
Write access to /var/www/html/glpioscarr/files/_lock has been validated.
Write access to /var/www/html/glpioscarr/files/_pictures has been validated.
Write access to /var/www/html/glpioscarr/files/_plugins has been validated.
Write access to /var/www/html/glpioscarr/files/_rss has been validated.
Write access to /var/www/html/glpioscarr/files/_sessions has been validated.
Write access to /var/www/html/glpioscarr/files/_tmp has been validated.
Write access to /var/www/html/glpioscarr/files/_uploads has been validated.
For security reasons, SELinux mode should be Enforcing.

Web server root directory configuration is not safe as it permits access to non-public files. See installation documentation for more details.
The following directories should be placed outside "/var/www/html/glpioscarr":
‣ "/var/www/html/glpioscarr/files" ("GLPI_VAR_DIR")
‣ "/var/www/html/glpioscarr/config" ("GLPI_CONFIG_DIR")
You can ignore this suggestion if your web server root directory is "/var/www/html/glpioscarr/public".
Sessions configuration is secured.
OS and PHP are relying on 64 bits integers.
exif extension is installed.
ldap extension is installed.
openssl extension is installed.
Following extensions are installed: bz2, Phar, zip.
Zend OPcache extension is installed.
Following extensions are installed: ctype, iconv, mbstring, sodium.
Write access to /var/www/html/glpioscarr/marketplace has been validated.
Timezones seems not loaded, see https://glpi-install.readthedocs.io/en/latest/timezones.html.

</pre></details><details><summary>GLPI constants</summary><pre> 
GLPI_ROOT: "/var/www/html/glpioscarr"
GLPI_CONFIG_DIR: "/var/www/html/glpioscarr/config"
GLPI_VAR_DIR: "/var/www/html/glpioscarr/files"
GLPI_MARKETPLACE_DIR: "/var/www/html/glpioscarr/marketplace"
GLPI_USE_CSRF_CHECK: "0"
GLPI_CSRF_EXPIRES: "7200"
GLPI_CSRF_MAX_TOKENS: "100"
GLPI_USE_IDOR_CHECK: "1"
GLPI_IDOR_EXPIRES: "7200"
GLPI_ALLOW_IFRAME_IN_RICH_TEXT: false
GLPI_SERVERSIDE_URL_ALLOWLIST: ["/^(https?|feed):\\/\\/[^@:]+(\\/.*)?$/"]
GLPI_TELEMETRY_URI: "https://telemetry.glpi-project.org"
GLPI_INSTALL_MODE: "TARBALL"
GLPI_NETWORK_MAIL: "glpi@teclib.com"
GLPI_NETWORK_SERVICES: "https://services.glpi-network.com"
GLPI_MARKETPLACE_ALLOW_OVERRIDE: true
GLPI_MARKETPLACE_MANUAL_DOWNLOADS: true
GLPI_USER_AGENT_EXTRA_COMMENTS: ""
GLPI_DISABLE_ONLY_FULL_GROUP_BY_SQL_MODE: "1"
GLPI_AJAX_DASHBOARD: "1"
GLPI_CALDAV_IMPORT_STATE: 0
GLPI_DEMO_MODE: "0"
GLPI_CENTRAL_WARNINGS: "1"
GLPI_DOC_DIR: "/var/www/html/glpioscarr/files"
GLPI_CACHE_DIR: "/var/www/html/glpioscarr/files/_cache"
GLPI_CRON_DIR: "/var/www/html/glpioscarr/files/_cron"
GLPI_DUMP_DIR: "/var/www/html/glpioscarr/files/_dumps"
GLPI_GRAPH_DIR: "/var/www/html/glpioscarr/files/_graphs"
GLPI_LOCAL_I18N_DIR: "/var/www/html/glpioscarr/files/_locales"
GLPI_LOCK_DIR: "/var/www/html/glpioscarr/files/_lock"
GLPI_LOG_DIR: "/var/www/html/glpioscarr/files/_log"
GLPI_PICTURE_DIR: "/var/www/html/glpioscarr/files/_pictures"
GLPI_PLUGIN_DOC_DIR: "/var/www/html/glpioscarr/files/_plugins"
GLPI_RSS_DIR: "/var/www/html/glpioscarr/files/_rss"
GLPI_SESSION_DIR: "/var/www/html/glpioscarr/files/_sessions"
GLPI_TMP_DIR: "/var/www/html/glpioscarr/files/_tmp"
GLPI_UPLOAD_DIR: "/var/www/html/glpioscarr/files/_uploads"
GLPI_INVENTORY_DIR: "/var/www/html/glpioscarr/files/_inventories"
GLPI_NETWORK_REGISTRATION_API_URL: "https://services.glpi-network.com/api/registration/"
GLPI_MARKETPLACE_PLUGINS_API_URI: "https://services.glpi-network.com/api/marketplace/"
GLPI_I18N_DIR: "/var/www/html/glpioscarr/locales"
GLPI_VERSION: "10.0.9"
GLPI_SCHEMA_VERSION: "10.0.9"
GLPI_MARKETPLACE_PRERELEASES: false
GLPI_MIN_PHP: "7.4.0"
GLPI_MAX_PHP: "8.3.0"
GLPI_YEAR: "2023"

</pre></details><details><summary>Libraries</summary><pre> 
htmlawed/htmlawed version 1.2.14 in (/var/www/html/glpioscarr/vendor/htmlawed/htmlawed)
phpmailer/phpmailer version 6.8.0 in (/var/www/html/glpioscarr/vendor/phpmailer/phpmailer/src)
simplepie/simplepie version 1.5.8 in (/var/www/html/glpioscarr/vendor/simplepie/simplepie/library)
tecnickcom/tcpdf version 6.6.2 in (/var/www/html/glpioscarr/vendor/tecnickcom/tcpdf)
michelf/php-markdown in (/var/www/html/glpioscarr/vendor/michelf/php-markdown/Michelf)
true/punycode in (/var/www/html/glpioscarr/vendor/true/punycode/src)
iamcal/lib_autolink in (/var/www/html/glpioscarr/vendor/iamcal/lib_autolink)
sabre/dav in (/var/www/html/glpioscarr/vendor/sabre/dav/lib/DAV)
sabre/http in (/var/www/html/glpioscarr/vendor/sabre/http/lib)
sabre/uri in (/var/www/html/glpioscarr/vendor/sabre/uri/lib)
sabre/vobject in (/var/www/html/glpioscarr/vendor/sabre/vobject/lib)
laminas/laminas-i18n in (/var/www/html/glpioscarr/vendor/laminas/laminas-i18n/src)
laminas/laminas-servicemanager in (/var/www/html/glpioscarr/vendor/laminas/laminas-servicemanager/src)
monolog/monolog in (/var/www/html/glpioscarr/vendor/monolog/monolog/src/Monolog)
sebastian/diff in (/var/www/html/glpioscarr/vendor/sebastian/diff/src)
donatj/phpuseragentparser in (/var/www/html/glpioscarr/vendor/donatj/phpuseragentparser/src/UserAgent)
elvanto/litemoji in (/var/www/html/glpioscarr/vendor/elvanto/litemoji/src)
symfony/console in (/var/www/html/glpioscarr/vendor/symfony/console)
scssphp/scssphp in (/var/www/html/glpioscarr/vendor/scssphp/scssphp/src)
laminas/laminas-mail in (/var/www/html/glpioscarr/vendor/laminas/laminas-mail/src/Protocol)
laminas/laminas-mime in (/var/www/html/glpioscarr/vendor/laminas/laminas-mime/src)
rlanvin/php-rrule in (/var/www/html/glpioscarr/vendor/rlanvin/php-rrule/src)
blueimp/jquery-file-upload in (/var/www/html/glpioscarr/vendor/blueimp/jquery-file-upload/server/php)
ramsey/uuid in (/var/www/html/glpioscarr/vendor/ramsey/uuid/src)
psr/log in (/var/www/html/glpioscarr/vendor/psr/log/Psr/Log)
psr/simple-cache in (/var/www/html/glpioscarr/vendor/psr/simple-cache/src)
psr/cache in (/var/www/html/glpioscarr/vendor/psr/cache/src)
league/csv in (/var/www/html/glpioscarr/vendor/league/csv/src)
mexitek/phpcolors in (/var/www/html/glpioscarr/vendor/mexitek/phpcolors/src/Mexitek/PHPColors)
guzzlehttp/guzzle in (/var/www/html/glpioscarr/vendor/guzzlehttp/guzzle/src)
guzzlehttp/psr7 in (/var/www/html/glpioscarr/vendor/guzzlehttp/psr7/src)
glpi-project/inventory_format in (/var/www/html/glpioscarr/vendor/glpi-project/inventory_format/lib/php)
wapmorgan/unified-archive in (/var/www/html/glpioscarr/vendor/wapmorgan/unified-archive/src)
paragonie/sodium_compat in (/var/www/html/glpioscarr/vendor/paragonie/sodium_compat/src)
symfony/cache in (/var/www/html/glpioscarr/vendor/symfony/cache)
html2text/html2text in (/var/www/html/glpioscarr/vendor/html2text/html2text/src)
symfony/css-selector in (/var/www/html/glpioscarr/vendor/symfony/css-selector)
symfony/dom-crawler in (/var/www/html/glpioscarr/vendor/symfony/dom-crawler)
twig/twig in (/var/www/html/glpioscarr/vendor/twig/twig/src)
twig/string-extra in (/var/www/html/glpioscarr/vendor/twig/string-extra)
symfony/polyfill-ctype not found
symfony/polyfill-iconv not found
symfony/polyfill-mbstring not found
symfony/polyfill-php80 not found
symfony/polyfill-php81 not found
symfony/polyfill-php82 in (/var/www/html/glpioscarr/vendor/symfony/polyfill-php82)
league/oauth2-client in (/var/www/html/glpioscarr/vendor/league/oauth2-client/src/Provider)
league/oauth2-google in (/var/www/html/glpioscarr/vendor/league/oauth2-google/src/Provider)
thenetworg/oauth2-azure in (/var/www/html/glpioscarr/vendor/thenetworg/oauth2-azure/src/Provider)

</pre></details><details><summary>LDAP directories</summary><pre> 
Server: 'cnamts.local', Port: '389', BaseDN: 'OU=Utilisateurs,OU=116901-CPAM Rhone,OU=Assurance Maladie,DC=cnamts,DC=local',
		Connection filter: '(&(objectClass=user)(objectCategory=person)(!(userAccountControl:1.2.840.113556.1.4.803:=2)))', RootDN:
		'C116901-CONSULT_ADN@cnamts.local', Use TLS: none

</pre></details><details><summary>SQL replicas</summary><pre> 
Not active

</pre></details><details><summary>Notifications</summary><pre> 
Way of sending emails: PHP

</pre></details><details><summary>Plugins list</summary><pre> 
	formcreator          Name: Form Creator                   Version: 2.13.4     State: Enabled                                 
		Install Method: Manual

</pre></details>
```

### Steps To reproduce

add graph Ticket par statut SLA et par groupe de techniciens on dashboard and checked number of ticket from each group

**Comment by trasher** [comment]
Please try if you reproduce on latest stable release.

**Comment by EmilieBy** [comment]
in 10.010?

**Comment by cedric-anne** [comment]
@EmilieBy 

Could you try to apply the patch proposed in https://github.com/glpi-project/glpi/pull/15754/files to see if problem is solved?

**Comment by cedric-anne** [comment]
Fixed by #15754
